cypress/integration/LogDialogs/WhatsYourName.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+*/
+
+const models = require('../../support/Models')
+const modelPage = require('../../support/components/ModelPage')
+const logDialogPage = require('../../support/components/logdialogspage')
+
+describe("What's your name", () =>
```

Comment:
Would like to see comments at the beginning of each test summarizing what it it testing